module/VuFind/src/VuFind/Db/Entity/ExchangeArrayTrait.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Entity model for user table
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2023.
```

Comment:
Might as well use the current year:

```suggestion
 * Copyright (C) Villanova University 2025.
```